src/Samples/NetProjectionSample/SimpleMathProjection/SimpleMathProjection.csproj

```diff
@@ -5,6 +5,10 @@
     <TargetFramework>net6.0-windows10.0.19041.0</TargetFramework>
     <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
     <Platforms>x64;x86;ARM64</Platforms>
```

Comment:
Should we just set Platform to AnyCPU instead of using PlatformTarget? This seems to work as well

Reply:
If it works with the different configurations, I agree that is probably better, I think as it represents what the binary actually is.

Reply:
verified that this works, and we can set the Configuration Manager so that SimpleMathProjection project always builds with Platform AnyCPU for each solution architecture
![image](https://user-images.githubusercontent.com/69016571/152572177-1b2227a5-8c46-4c8f-aed9-d2b0fcdc8c30.png)